### DISCUSSION TITLE
Any idea how to flush/reset a database's table metadata?

### DISCUSSION THREAD

**Discussion by jeffcwolf** [dbgate/dbgate#873] [Q&A]
Hi, DBGate is telling me that it can't load an SQLite database because a table can't be found. But it can access the database directly using queries (and the database can be accessed just fine in other apps). SO there must be some cache that is saving the metadata about the the tables, and it's out of date. Any idea how to reset this? Refreshing the schema doesn't work. Disconnect, deleting, and re-establishing the connection to that database does not work either. Restarting DBGate does not work. The only thing I have not tried is re-installing it. What am I missing? Any ideas? Thanks!

**Comment by janproch** [comment]
Could you please share an exact error message? (or screenshot with error)

DbGate has cache of database structure, but after restarting DbGate all data from this cache is reloaded. Command Sync model (full) also reloads structure from scratch. (Refresh button in Tables view does incremental structure update)

**Comment by jeffcwolf** [comment]
The error is 'No such table:main.[name of an old table that I no longer have in the database]

I tried clicking on the Sync model (full) command (after clicking on the Error indicator at the bottom of the screen), but this did not correct the issue.

Any other ideas?
